src/openms/include/OpenMS/DATASTRUCTURES/QTCluster.h

```diff
@@ -367,6 +367,4 @@ namespace OpenMS
       bool finalized_;
   };
 
-  // needed for the heap
-  bool operator<(const QTCluster& q1, const QTCluster& q2);
```

Comment:
@cbielow this one failed with (ambigious method) after adding const to the unary operator< 
To me, it seems as it can be safely removed.

Reply:
yes. Since you added `const` to `bool operator<(const QTCluster& cluster) const;`, this now takes part in overload resolution and there should even be an error - or at least a warning, since `operator<` is now ambiguous.

Strangely, I dont know when `bool operator<(const QTCluster& cluster);` would ever have been called...

Reply:
after removal, the unary is now called when adding lazy to the cluster (which previously called the binary operator).